src/components/availability-label/availability-label.dev.tsx

```diff
@@ -42,19 +47,29 @@ const Template: ComponentStory<typeof AvailabilityLabel> = (
 ) => <AvailabilityLabel {...args} />;
 
 export const Available = Template.bind({});
+Available.args = {
+  materialId: ["61435867"]
+};
+
+export const MoreThanOneID = Template.bind({});
+MoreThanOneID.args = {
+  materialId: ["62523611", "62150041", "61435867"]
+};
 
 export const Selected = Template.bind({});
 Selected.args = {
   manifestText: "lydbog (cd-mp3)",
   availabilityText: "udlånt",
-  state: "selected",
-  link: undefined
+  selected: true
 };
 
 export const Unavailable = Template.bind({});
 Unavailable.args = {
   manifestText: "ebog",
-  availabilityText: "online",
-  state: "unavailable",
-  link: undefined
+  availabilityText: "online"
+};
+
+export const harryPotter = Template.bind({});
+harryPotter.args = {
+  materialId: ["62523611"]
 };
```

Comment:
Please remove this story.

I do not see a reason for it to be added. It does not seem to add anything to the stories already available.